code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1009,6 +1009,39 @@
 	if(mind.assigned_role.title in SSjob.name_occupations)
 		.[mind.assigned_role.title] = minutes
 
+/mob/living/carbon/human/can_be_absorbed(datum/antagonist/changeling/absorbing_changeling, verbose = TRUE, is_true_absorb = FALSE)
+	var/mob/changeling_mob = absorbing_changeling?.owner?.current
+
+	if(!istype(dna) || (NO_DNA_COPY in dna.species.species_traits))
+		if(verbose)
```

Comment:
Remember to add to the if check whether the mob exists as well, since `to_chat(null, text)` runtimes.